src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -2426,5 +2426,43 @@ static void M()
 
             await TestAsync(markup, expectedOrderedItems);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        [CompilerTrait(CompilerFeature.FunctionPointers)]
+        public async Task TestFunctionPointer()
+        {
+            var markup = @"
+class C
+{
+    unsafe static void M()
+    {
+        delegate*<int, int> functionPointer;
+        functionPointer($$);
+    }
+}";
+
+            var expectedOrderedItems = new List<SignatureHelpTestItem> { new SignatureHelpTestItem("int (int)", currentParameterIndex: 0) };
```

Comment:
If we want to do something to make this be more clear it's a function pointer I'm OK with that. I didn't find anything I liked though. `int delegate*<int, int>(int)` is reundant and lists the parameters twice, `int delegate*<>(int)` felt like I was making up some syntax, and `int (function pointer)(int)` felt like Lisp.

Reply:
It would be nice to make it more obvious its a function pointer. Does `int delegate*(int)` hide too much info?

Reply:
Actually I think I like that. Sounds good. 😄